### PR TITLE
PR for snowflake-wiz vhol

### PR DESCRIPTION
**Pull request by guy-mast-wiz**
first commit

### CODE REVIEW

File: site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
+author: Guy Mast 
+id: security_analytics_with_wiz_and_snowflake
+summary: learn about the Wiz and Snowflake integration for analyzing Wiz data in Snowflake
+categories: cybersecurity,solution-examples,partner-integrations
+environments: web
+status: Hidden
```
Comment (iamontheinet) [inline site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: @guy-mast-wiz Change the status to Published.

File: site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+
+### Drop Role
+```sql
+DROP ROLE IF EXISTS CLOUD_SECURITY_ANALYST_ROLE;
+```
+This command deletes the CLOUD_SECURITY_ANALYST_ROLE role.
+
+
+
+## Conclusion & Next Steps
```
Comment (iamontheinet) [inline site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: @guy-mast-wiz Change the label to "Conclusion And Resources" and make the format, layout, and content similar to https://quickstarts.snowflake.com/guide/getting_started_with_snowflake_arctic/index.html#5

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md (hunk 1 above)
Comment (guy-mast-wiz) [reply site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: Sure will do, how can I make this first hidden and not visible on https://quickstarts.snowflake.com/?

Comment (iamontheinet) [reply site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: What's the reason behind doing that? If it's not ready to be published, then let's wait to merge the PR.

Comment (guy-mast-wiz) [reply site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: please ignore this, i have updated the status as you requested

File: site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md (hunk 2 above)
Comment (guy-mast-wiz) [reply site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md]: Done

**Pull request merged**